pkgs/development/python-modules/htmllistparse/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, beautifulsoup4
+, html5lib
+, requests
+, fusepy
+}:
+
+buildPythonPackage rec {
+  pname = "htmllistparse";
+  version = "0.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-bcimvwPIQ7nTJYQ6JqI1GnlbVzzZKiybgnFiEBnGQII=";
+  };
+
+  propagatedBuildInputs = [ beautifulsoup4 html5lib requests fusepy ];
+
```

```suggestion

  # upstream has no tests
  doCheck = false;
  
  pythonImportsCheck = [
    "htmllistparse"
  ];

```
Please enable appropriate tests.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

---

wouldn't `doCheck = false` prevent the pythonImportsCheck from doing anything?